v3/integrations/nrpgx5/nrpgx5.go

```diff
@@ -64,10 +64,9 @@ type (
 )
 
 const (
-	querySegmentKey   nrPgxSegmentType = "nrPgx5Segment"
-	prepareSegmentKey nrPgxSegmentType = "prepareNrPgx5Segment"
-	batchSegmentKey   nrPgxSegmentType = "batchNrPgx5Segment"
-	querySecurityKey  nrPgxSegmentType = "nrPgx5SecurityToken"
+	querySegmentKey  nrPgxSegmentType = "nrPgx5Segment"
+	batchSegmentKey  nrPgxSegmentType = "batchNrPgx5Segment"
+	querySecurityKey nrPgxSegmentType = "nrPgx5SecurityToken"
```

Comment:
Are these three lines showing as added because of a linter/formatting that was applied?